modules/ps_facetedsearch/views/templates/front/catalog/facets.tpl

```diff
@@ -177,23 +189,24 @@
                       data-slider-specifications="{$facet.properties.specifications|@json_encode}"
                       data-slider-encoded-url="{$filter.nextEncodedFacetsURL}"
                       data-slider-direction="{$language.is_rtl}"
-                   >
+                  >
                     </div>
                   <input 
                     type="hidden"
                     class="form-range-start js-faceted-slider js-faceted-slider-start"
                     id="slider-range_{$_expand_id}-start"
-                   >
+                  >
                   <input 
                     type="hidden"
                     class="form-range-start js-faceted-slider js-faceted-slider-end"
                     id="slider-range_{$_expand_id}-end"
-                   >
-                 </div>
+                  >
+                </div>
                 {/foreach}
               {/block}
             {/if}
           </div>
+          {if !$smarty.foreach.facets.last}<hr class="my-0">{/if}
```

Comment:
```suggestion
```

Reply:
Why? This is needed

Reply:
horizontal lines on menu?
it seems somehow old fashioned!

Reply:
It's used to split every facets with a line, this comes from the mockup